### PR TITLE
Present medical demo

### PR DESCRIPTION
**Pull request by mosheber**

### CODE REVIEW

File: fastrag/readers/__init__.py [hunk 1]
```diff
@@ -3,8 +3,11 @@
 from haystack.nodes import BaseGenerator, BaseReader
 from haystack.schema import Document, MultiLabel
 
-from fastrag.readers.FiD import FiDReader
-from fastrag.readers.T5 import T5Reader
+# Currently removing these classes due to incompatibility with the Seq2SeqGenerator
+# Future versions will fix this discrepancy
+
+# from fastrag.readers.FiD import FiDReader
+# from fastrag.readers.T5 import T5Reader
```
Comment (peteriz) [inline fastrag/readers/__init__.py:10]: what did you change the version? I dont see you touched `setup.cfg`

File: fastrag/ui/prompt_llm.py [hunk 2]
```diff
@@ -24,6 +24,15 @@
 from fastrag.ui.utils import API_ENDPOINT, display_runtime_plot, haystack_is_ready, query
 
 DEFAULT_PROMPT = """Answer the question using the provided context. Your answer should be in your own words and be no longer than 50 words. \n\n Context: {join(documents)} \n\n Question: {query} \n\n Answer:"""
+pipeline_path = os.getenv("PIPELINE_PATH", None)
```
Comment (peteriz) [inline fastrag/ui/prompt_llm.py:27]: what is PIPELINE_PATH?

File: fastrag/readers/__init__.py (hunk 1 above)
Comment (mosheber) [reply fastrag/readers/__init__.py:10]: I updated the farm-haystack version. I re-launched the demo as well with the new version to ensure it works, and it does

File: fastrag/ui/prompt_llm.py (hunk 2 above)
Comment (mosheber) [reply fastrag/ui/prompt_llm.py]: I had it previously to change the default prompt used in the UI, but its not crucial. I removed it

**Pull request closed without merging**